src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1494,6 +1494,12 @@ export default class AircraftModel {
                 if (elapsed > this.taxi_time) {
                     this.setFlightPhase(FLIGHT_PHASE.WAITING);
                     this.moveToRunway(runwayModel);
+                    speech_say([
+                        { type: 'text', content: AirportController.airport_get().radio.twr },
+                        { type: 'callsign', content: this },
+                        { type: 'text', content: `, holding short runway ${runwayModel.getRadioName()}` }
+                    ]);
+                    UiController.ui_log(`${AirportController.airport_get().radio.twr}, ${this.callsign}, holding short runway ${runwayModel.name}`);
```

Comment:
Not really pretty.

@n8rzz Any thoughts?

Reply:
agreed, not pretty, but not bad either.  two things here:
- I just had to look up `.speech_say()` and, holy crap, we need to re-work that one.  yikes!!
- I'd set the string actually being logged (LN#1502) to a constant first and then simply pass it on to `.ui_log()`

`.speech_say()` is super gross, but that's not this method's fault.  added #1204 to account for that future work